### ISSUE TITLE
FZY doesn't work in the command line on WSL ConEmu

### ISSUE THREAD

**Issue by lylejantzi3rd** [jhawthorn/fzy#84]
I'm running Ubuntu 16.04.03 LTS under WSL on windows 1709 "Fall Creators Update." The terminal I'm using is ConEmu 171217 and the shell I'm using is ZSH.

I have fzy installed on the system. Nothing happens when I use it on the command line. It sits there and looks like it's waiting for input until I hit ctrl+c. I can type but nothing ever shows up on the screen. What's interesting is that fzy works fine inside the vim-picker plugin. I open vim and activate the plugin (which runs the system installed fzy) and the dropdown of results appears as expected.

**Comment by neuschaefer** [comment]
>  It sits there and looks like it's waiting for input until I hit ctrl+c.

Yes, that's what it does. When fzy starts, it reads lines from standard input until it reaches the end of file, and then it starts the user interface that allows interactive selection of a line.

Try feeding it something on stdin:

```
$ fzy
asdf
1234
foo
<CTRL-d to send EOF>
<the UI should start>
```

or

```
$ fzy < textfile.txt
<the UI should start immediately>
```

or

```
$ find /some/directory | fzy
<the UI should start once find is done>
```

**Comment by lylejantzi3rd** [comment]
Yeah, that was the issue. Thanks for being patient with me!
